lib/services/geocoding.js

```diff
@@ -33,6 +33,7 @@ function roundTo(value, places) {
  * @param {Array} options.bbox a bounding box argument: this is
  * a bounding box given as an array in the format [minX, minY, maxX, maxY].
  * Search results will be limited to the bounding box.
+ * Specify the language to use for response text and, for forward geocoding, query result weighting. Options are IETF language tags comprised of a mandatory ISO 639-1 language code and optionally one or more IETF subtags for country or script. More than one value can also be specified, separated by commas. 
```

Comment:
Thanks for catching that! Added in https://github.com/mapbox/mapbox-sdk-js/pull/185/commits/b607c5cae45b6f5463a4780f51f75602ef26c178.